src/components/study/study-details.tsx

```diff
@@ -32,60 +60,15 @@ export const StudyDetails: FC<{ studyId: string }> = ({ studyId }) => {
                         <Text>{study.piName}</Text>
                         <Text>{study.researcherName}</Text>
                         <Text>
-                            {study.descriptionDocPath && (
-                                <Badge
-                                    key={study.descriptionDocPath}
-                                    color="#D4D1F3"
-                                    c="black"
-                                    component="a"
-                                    href={studyDocumentURL(
-                                        study.id,
-                                        StudyDocumentType.DESCRIPTION,
-                                        study.descriptionDocPath,
-                                    )}
-                                    target="_blank"
-                                    rightSection={<Download />}
-                                    style={{ cursor: 'pointer' }}
-                                >
-                                    {study.descriptionDocPath}
-                                </Badge>
-                            )}
+                            {study.descriptionDocPath &&
+                                renderBadge(study.descriptionDocPath, StudyDocumentType.DESCRIPTION, study.id)}
```

Comment:
love this abstraction, how about taking it one step further and making `renderBadge` be a component that accepts a path that may be a string or null / undefined.  if it's not present it renders null, otherwise the same thing it renders now.  then this could all be:
```
<BadgeWithDescription path={study.descriptionDocPath} type={StudyDocumentType.DESCRIPTION} studyId={study.id} />
```